designs/administrator/4.5/alertmanager-config/index.md

```diff
@@ -0,0 +1,26 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Alertmanager Configuration
+
+![](img/alertmanager-first-time.png)
+- The default alertmanager configuration will ship with 3 receivers, two of which need to be configured to correctly function.
+
+![](img/alertmanager-editing-default.png)
```

Comment:
Minor nit, but wondering if "Label name" should actually be "Label key"?

Reply:
I initially used "Name" because I looked through the console and it seemed like we use Name in most places (environment variables, for instance). I was curious about whether there was a difference between these words and it seems like there's a subtlety that I had missed out on. "Key/Value" is generally used in the context of a database or hash table while "Name/Value" is used in "Category/Attribute" situations like the ones that appear throughout the console. With that in mind, I think we should stick with "Name" here and I think we should probably change the few instances of "Key" in the console to "Name" when appropriate

Reply:
Gotcha, thanks for looking into that. Makes sense to me 👍